src/bindings/python/docs/test_examples.md

```diff
@@ -3,17 +3,27 @@
 #### Building and environment
```

Comment:
```suggestion
## Build the environment
```

Reply:
This section actually isn't about `build the environment`. It's about how-to build and setup environment